From: Coly Li <bosong.ly@taobao.com>
Subject: remove redundant sense_buffer member in pmcraid_cmd
Date: Fri Dec  9 14:20:29 CST 2011
Patch-mainline: in-house
Reference:

This patch remove a redundant sense_buffer member defined in struct
pmcraid_cmd. This error broke RHEL 2.6.32-220 kernel compiling.

Signed-off-by: Coly Li <bosong.ly@taobao.com>
---
Index: linux-2.6.32-220.0.0.el5/drivers/scsi/pmcraid.h
===================================================================
--- linux-2.6.32-220.0.0.el5.orig/drivers/scsi/pmcraid.h
+++ linux-2.6.32-220.0.0.el5/drivers/scsi/pmcraid.h
@@ -578,7 +578,6 @@ struct pmcraid_cmd {
 	struct pmcraid_control_block *ioa_cb;
 	dma_addr_t ioa_cb_bus_addr;
 	dma_addr_t dma_handle;
-	u8 *sense_buffer;
 
 	/* pointer to mid layer structure of SCSI commands */
 	struct scsi_cmnd *scsi_cmd;
